index.html

```diff
@@ -223,23 +221,25 @@ <h4>Skills</h4>
                 </ul>
             </div>
         </div>
-        <div class="get-started">
+        <div class="get-started" id="get-started">
```

Comment:
### OPTIONAL
- [ ] Kindly adjust the getting started and footer section to fill the entire width of the page, currently this is how it looks ⬇️ 

![Screenshot_27](https://user-images.githubusercontent.com/69638013/161210209-f5d7bf48-d6bc-4348-b5ef-99740287cc9b.png)
- Doing this will make your design look similar to the figma design 👍🏿

Reply:
Hey @prolajumokeoni 

Thanks for the tip. I suspect I'll have to modify the CSS to have it compatible with different browsers. Getting on it.

Reply:
Ok great